source/core/slang-io.cpp

```diff
@@ -17,6 +30,8 @@
 
 namespace Slang
 {
+    static String s_executablePath;
```

Comment:
This appears to get declared but not used, since the `GetExecutablePath()` function ends up using a function-`static` variable instead.